lib/pharos/config_schema.rb

```diff
@@ -33,10 +33,18 @@ def self.messages
           optional(:endpoint).filled(:str?)
         end
         optional(:network).schema do
+          optional(:provider).filled(included_in?: %(weave calico))
           optional(:dns_replicas).filled(:int?, gt?: 0)
           optional(:service_cidr).filled(:str?)
           optional(:pod_network_cidr).filled(:str?)
-          optional(:trusted_subnets).each(type?: String)
+          # TODO: deprecate trusted_subnets
```

Comment:
Need to figure out `Dry::Struct` validations to make it either reject old `config.network.trusted_subnets`, or warn about it and default the new `config.network.weave.trusted_subnets` to the deprecated field's value.

Reply:
```
==> Reading instructions ...
==> Invalid configuration file:
---
:network:
  :trusted_subnets:
  - cannot be defined
```